src/hooks/useNotifications.tsx

```diff
@@ -82,24 +97,41 @@ const notificationQuery = gql`
               }
             }
           }
+          ... on PrivatePostNotification {
+            post {
+              id
+              __typename
+              ... on ActivePost {
+                authorId
+              }
+            }
+          }
         }
       }
     }
   }
 `;
 
-const supportedNotificationTypes = [
+export type SupportedNotifications = [
   'PostReplyNotification',
   'CircleAdminPostNotification',
   'SurveyAssignedNotification',
+  'PrivatePostNotification',
 ];
+
+function isSupportedNotification(
+  __typename: any,
+): __typename is SupportedNotifications {
+  return true;
+}
+
 const selectNotifications = (
   data: NotificationQueryResponse,
 ): NotificationQueryResponse => {
   return {
     notificationsForUser: {
       edges: data.notificationsForUser.edges.filter((edge) =>
```

Comment:
Are notifications sorted by most recent? If not that would be nice so the most recent message would show up at the top of the list

Reply:
Good question, these might be loaded earliest to latest which would result in the newest unread message at the bottom of the list. I will follow up on this in another PR.